http_service/bugbug_http/boot.py

```diff
@@ -97,10 +97,10 @@ def extract_commit_experiences() -> None:
             utils.extract_file(os.path.join("data", repository.COMMIT_EXPERIENCES_DB))
             logger.info("Commit experiences DB extracted.")
         except FileNotFoundError:
+            assert ALLOW_MISSING_MODELS
```

Comment:
Can you make this change in a separate PR?

Reply:
Done https://github.com/mozilla/bugbug/pull/1720